gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/Datastore.java

```diff
@@ -105,4 +104,40 @@
    * Returns a new KeyFactory for this service
    */
   KeyFactory newKeyFactory();
+
+  /**
+   * Returns an {@link Entity} for the given {@link Key} or {@code null} if it doesn't exist.
+   * {@link ReadOption}s can be specified if desired.
+   *
+   * @throws DatastoreException upon failure
+   */
+  Entity get(Key key, ReadOption... options);
+
+  /**
+   * Returns an {@link Entity} for each given {@link Key} that exists in the Datastore.
+   * The order of the result is unspecified.
+   * Results are loaded lazily, so it is possible to get a {@code DatastoreException}
+   * from the returned {@code Iterator}'s {@link Iterator#hasNext hasNext} or
```

Comment:
I think there's a (small) benefit here because it's nice to specify "the returned Iterator's", and if we don't change the label, the word Iterator will show up 3 times in the javadoc in quick succession.